Sources/SwifterSwift/Foundation/DecimalExtensions.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  DecimalExtensions.swift
+//  SwifterSwift
+//
+//  Created by viktart on 04/09/2023.
+//  Copyright © 2023 SwifterSwift
+//
+
+import Foundation
+
+public extension Decimal {
+    var doubleValue: Double {
+        return NSDecimalNumber(decimal: self).doubleValue
+    }
```

Comment:
No need for `return`, and you can just cast directly.
```suggestion
    var doubleValue: Double { (self as NSDecimalNumber).doubleValue }
```